strings/find_anagram_indices.py

```diff
@@ -0,0 +1,81 @@
+from collections import Counter, defaultdict
+
+
+def find_anagram_indices(word: str, letters: str) -> list:
```

Comment:
Please provide the generic parameters for the list type and use `List[type]` from the `typing` module, import it using `from typing import List`.